neuraxle/hyperparams/space.py

```diff
@@ -226,6 +228,76 @@ class HyperparameterSamples(RecursiveDict):
     def __init__(self, *args, separator=None, **kwds):
         super().__init__(*args, separator=separator, **kwds)
 
+    def compress(self) -> 'CompressedHyperparameterSamples':
+        """Compresses the HyperparameterSamples representation."""
+        return CompressedHyperparameterSamples(self)
+
+
+@dataclass
+class CompressedHyperparameter:
+    """CompressedHyperParameter"""
```

Comment:
```suggestion
    """
    Compressed and easy to read representation of a hyperparameter sample.
    This class is used by the HyperparameterSamples class.
    """
```

Reply:
This fixes the Codacy status check.

Reply:
Thanks, i will keep this format in mind. I was trying different formats, to pass the check. :)